engine/src/main/java/org/terasology/rendering/logic/MeshRenderer.java

```diff
@@ -203,8 +202,7 @@ private void renderEntitiesByMaterial(SetMultimap<Material, EntityRef> meshByMat
                     location.getWorldPosition(worldPos);
                     float worldScale = location.getWorldScale();
 
-                    javax.vecmath.Matrix4f matrixWorldSpace = new javax.vecmath.Matrix4f(VecMath.to(worldRot), VecMath.to(worldPos), worldScale);
-                    transWorldSpace.set(matrixWorldSpace);
+                    Transform transWorldSpace = new Transform(worldPos, worldRot, worldScale);
```

Comment:
Could this variable be renamed `toWorldSpace`?

Reply:
I see what you are getting at, but I think `worldSpaceTransform` would arguably be a better name, since transforms usually represent a single space (in this case, world space) to transform to/from.

Reply:
Hmmmm... I don't think I agree with that description. 

A transformation is a mapping from one space to another. Sure, you can -interpret- the information stored in the transformation as creating a new space relative to some other frame of reference, i.e. the world space. But the way we use it in our context is to transform vectors from one space to another. Crucially, a specific transformation brings you from space A to space B but it then needs to be inverted to go the other way around. So, the name of a variable storing a transformation in my opinion should reflect the direction of the transformation.

Reply:
Your description does make more sense the more I think about it. However, I still feel that `toWorldSpace` is a bit lacking as a variable name - to me it sounds more like a method. How about `toWorldSpaceTransform`? I don't normally agree with the practice of including the type in the variable name, but I feel it is justified in this case.

Reply:
Sure, I can live with that.